src/containers/DataDisplay/Filters/DataViewSearchableList.tsx

```diff
@@ -32,13 +32,13 @@ export const DataViewSearchableList: React.FC<SearchableListProps> = ({
   const [moreResults, setMoreResults] = useState(false)
   const [debounceOutput, setDebounceInput] = useDebounce(searchText)
 
-  const { column, code, searchFields, delimiter } = options
+  const { column, code, ...otherOptions } = options
```

Comment:
These properties (plus the `includeNull` prop) are not actually used in the front-end, but passed back unchanged to server with search queries, so no need to unpack them. And it means if we add more "filter_parameters" we won't need to deal with them here (unless they're relevant to front-end)